### PR TITLE
Fix/Update install process

### PR DESCRIPTION
**Pull request by JamesKEbert**
# Summary of Changes

This is a replacement / followup for #80 by @NeilSMyers.
This PR adds:
* Fixes the Aries Hooks node_modules issues
* Fixes multiple login issues
* Updates the installation & setup documentation
* Updates to AFJ 0.1.0-alpha.273

# Related Issues

N/A

# Pull Request Checklist

This is just a reminder about the most common mistakes. Please make sure that you tick all _appropriate_ boxes. But please read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!

If an item doesn't apply to your pull request, **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this).
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components.
- [x] Run prettier: `npm run style-format`
- [x] Updated **documentation** for changed code and new or modified features.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

_PR template adapted from the Python attrs project._

### CODE REVIEW

**Review by manghuli — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -27,122 +25,76 @@ iOS targets iOS 10.0+. Aries Bifold can only be run on physical devices as of ri
...
-### Prerequistes
-
-#### React Native
+Aries Bifold utilizes the [Aries Agent Test Harness (AATH)](https://github.com/hyperledger/aries-agent-test-harness) to test for Aries Agent compatibility and interoperability.
 
-Installation instructions are documented [here](https://reactnative.dev/docs/environment-setup).
+Aries Bifold also utilizes the [Aries Toolbox](https://github.com/hyperledger/aries-toolbox) & [aries-acapy-plugin-toolbox](https://github.com/hyperledger/aries-acapy-plugin-toolbox) for issuer and verifier roles.
 
-Troubleshooting guides can be found in the [docs/installation](./docs/INSTALLATION.md) directory.
+## Install
```
Comment (karimStekelenburg) [inline README.md:32]: This section contains guides on installation, configuration and getting the agent running. I think a more common and familiar structure to most devs is having individual `install`, `configure` and `run` sections.

File: README.md [hunk 2]
```diff
@@ -27,122 +25,76 @@ iOS targets iOS 10.0+. Aries Bifold can only be run on physical devices as of ri
...
+        * (Android)
+            ```sh
+            npm run android
+            ```
+        * (iOS)
+            ```sh
+            npm run ios
+            ```
+    * (iOS) Via Xcode:
+        Choose your physical iOS device as the destination. Click the "Play" button to Build and Run.
```
Comment (karimStekelenburg) [inline README.md:79]: Maybe it's an idea to create an additional section/quote (something that stands out) that states Bifold currently doesn't work in the iOS simulator and thus requires a physical iOS device? That might avoid some confusion for devs getting started with Bifold.

File: README.md (hunk 1 above)
Comment (JamesKEbert) [reply README.md:32]: Agreed--altered and pushed!

**Review by karimStekelenburg — APPROVED** [review]
Nice! Good to go :)

**Pull request merged**